#[test]
fn day_01_part_1() -> Result<(), Box<dyn std::error::Error>> {
    todo!();
}

#[test]
fn day_01_part_2() -> Result<(), Box<dyn std::error::Error>> {
    todo!();
}
